src/lib/UniversalDiscovery/Event/Subscriber/ImageAssetAllowedContentTypes.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\UniversalDiscovery\Event\Subscriber;
+
+use EzSystems\EzPlatformAdminUi\UniversalDiscovery\Event\ConfigResolveEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use eZ\Publish\Core\FieldType\ImageAsset\AssetMapper;
+
+class ImageAssetAllowedContentTypes implements EventSubscriberInterface
```

Comment:
```suggestion
final class ImageAssetAllowedContentTypes implements EventSubscriberInterface
```

IMHO it's worth to add unit test coverage for this class :wink: